src/DataFixtures/ContentFixtures.php

```diff
@@ -104,6 +104,24 @@ private function loadContent(ObjectManager $manager): void
                     }
                 }
 
+                foreach ($contentType['taxonomy'] as $taxonomySlug) {
+                    if ($taxonomySlug === 'categories') {
+                        $amount = 2;
+                    } elseif ($taxonomySlug === 'tags') {
+                        $amount = 4;
+                    } else {
+                        $amount = 1;
+                    }
+
+                    for ($i = 1; $i <= $amount; $i++) {
```

Comment:
let's use only one of `for` or `foreach(range)`

Reply:
will do.

Reply:
If i replace this with `foreach (range(1, $amount) as $number) {`, cs-fix complains about an unused variable `$number`. Not sure how to fix that without ugly hacks. 🤔

Reply:
then use for but everywhere :)